src/api/models/Activity.ts

```diff
@@ -118,6 +97,11 @@ Activity.init(
         },
         endTime: {
             type: DataTypes.DATE
+        },
+        address: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            defaultValue: 'Nowhere'
```

Comment:
Will it throw an error if there is no 'address' attribute in the request or will it populate it with this default value silently?
I think it's better to throw an error rather than having a default value here

Reply:
+1